components/bridge/Deposit.tsx

```diff
@@ -30,7 +31,7 @@ const Deposit: React.FC<{
 	const { updateBalances } = useWallet();
 
 	const resetModal = () => {
-		setModal({ state: "", text: "", hash: "" });
+		setModal({ state: "", text: "", hash: "", title: "" });
 		setModalOpen(false);
```

Comment:
if we want to reset the modal, shouldn't we just do `setModal(null)`?

Reply:
should be fixed now